lib/common/events.ts

```diff
@@ -91,8 +91,9 @@ export function patchEventTarget(
 
   // global shared zoneAwareCallback to handle all event callback with capture = false
   const globalZoneAwareCallback = function(event: Event) {
-    const target = this || _global;
-
+    // event.target is needed for Samusung TV and SourceBuffer
+    // || global is needed https://github.com/angular/zone.js/issues/190
+    const target: any = this || event && event.target || _global;
```

Comment:
@mhevery , should we add 
```javascript
if (!event) {
  return;
} 
```

because in the next line, we use `event.type` directly.